src/cargo/util/command_prelude.rs

```diff
@@ -708,6 +708,7 @@ Run `{cmd}` to see possible targets."
         build_config.build_plan = self.flag("build-plan");
         build_config.unit_graph = self.flag("unit-graph");
         build_config.future_incompat_report = self.flag("future-incompat-report");
+        build_config.checksum_freshness = self.flag("checksum-freshness");
```

Comment:
I think there might be a misunderstanding which makes me wonder how the tests are working. `self.flag` is for reading CLI flags from clap but no `checksum-freshness` CLI flag was created.  Instead there is `checksum-freshness` unstable feature flag that should be accessible as `gctx.cli_unstable().checksum_freshness`

Reply:
Resolved in 6cf92e1f1677ebe030ddfabea8782e2f721378ce, this was just vestigial.